doc/lightning-listpeers.7.md

```diff
@@ -100,6 +100,8 @@ The objects in the *channels* array will have at least these fields:
   a number followed by a string unit.
 * *total\_msat*: A string describing the total capacity of the channel;
   a number followed by a string unit.
+* *fee_base*: The applied routing fees for this half channel as msat string.
```

Comment:
The term "half channel" is never otherwise used in this, or any other, doc.  Perhaps better "the routing fee we will charge for forwards going out over this channel, regardless of payment size".